backend/src/akvo/lumen/lib/import/flow_common.clj

```diff
@@ -102,14 +102,31 @@
 ;; {question-group-id -> [{question-id -> response}]
 ;; to
 ;; {question-id -> first-response}
-(defn question-responses
-  "Returns a map from question-id to the first response iteration"
-  [questions responses]
+(defn- question-responses-base [questions responses]
   (->> responses
        vals
        (map first)
        (apply merge)))
 
+;; Transforms the structure
+;; {question-group-id -> [{question-id -> response}]
+;; to
+;; [(with-meta
+;;          {question-id -> first-response}
+;;          {:ns xxx})]
+(defn question-responses
+  "Returns a list of maps with meta from question-id to the first response iteration"
+  [groups questions responses]
+  (let [dict (let [[rep-col non-rep-col] (split-with :repeatable groups)]
```

Comment:
`split-with` is very suspicious.

Reply:
fixed with https://github.com/akvo/akvo-lumen/pull/2784/commits/bcb790ade8d1ecd0e3c08caca5acff3d7e7fac31